pkg/combustion/templates/13-embedded-registry.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -euo pipefail
+
+mkdir -p /etc/hauler
+
+mv haul.tar.zst /etc/hauler/haul.tar.zst
+mv hauler /usr/bin/hauler
+
+cat <<- EOF > /etc/systemd/system/registry.service
+  [Unit]
+  Description=Hauler Load and Serve
+  After=network.target
+
+  [Service]
+  Type=simple
+  User=root
+  WorkingDirectory=/etc/hauler
+  ExecStartPre=/usr/bin/hauler store load haul.tar.zst
+  ExecStart=/usr/bin/hauler store serve
+  Restart=on-failure
+
+  [Install]
+  WantedBy=multi-user.target
+EOF
+
+systemctl enable registry.service
```

Comment:
Can we call this "local-registry" or something? I think "registry" is a little vague.

Reply:
That's a good call. I'd go a step further and make sure we include some sort of designation that shows it came from EIB, since `local-registry` still won't give people a ton of hints where it came from.

Reply:
Yeah I'm a little hesitant to suggest that we have "eib" in there, but I'll go with the majority here if we think it needs to have it.